service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersRemovedEventHandlerTest.java

```diff
@@ -151,6 +151,56 @@ void shouldSendNotification() {
             CASE_ID);
     }
 
+    @Test
+    void shouldNotSendNotificationToCourtIfCourtEmailIsNull() {
+        final HearingOrder additionalOrder = HearingOrder.builder().type(HearingOrderType.DRAFT_CMO).build();
+
+        final List<Element<HearingOrder>> caseManagementOrdersBefore = Stream.of(ORDER_TO_BE_REMOVED,
+                element(OTHER_ORDER_ID, additionalOrder))
+            .collect(Collectors.toList());
+
+        final HearingOrdersBundle hearingOrdersBundleBefore = HearingOrdersBundle.builder()
+            .hearingId(HEARING.getId())
+            .orders(caseManagementOrdersBefore).build();
+
+        final CaseData caseDataBefore = CaseData.builder()
+            .id(CASE_ID)
+            .allocatedJudge(allocatedJudge())
+            .hearingOrdersBundlesDrafts(List.of(
+                element(HEARING_ORDER_BUNDLE_ID, hearingOrdersBundleBefore)
+            ))
+            .hearingDetails(List.of(HEARING))
+            .build();
+
+        final List<Element<HearingOrder>> caseManagementOrdersAfter = List.of(element(OTHER_ORDER_ID, additionalOrder));
+
+        when(draftOrdersRemovedContentProvider.buildContent(
+            caseDataBefore, Optional.of(HEARING.getValue()), JUDGE, ORDER_TO_BE_REMOVED.getValue(), REMOVAL_REASON)
+        ).thenReturn(DRAFT_ORDERS_REMOVED_TEMPLATE_DATA);
+        when(courtService.getCourtEmailNotCtsc(any())).thenReturn(null);
+        when(localAuthorityRecipients.getRecipients(any())).thenReturn(Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS));
+
+        final CaseData caseDataAfter = caseDataBefore.toBuilder()
+            .hearingOrdersBundlesDrafts(List.of(
+                element(HEARING_ORDER_BUNDLE_ID,
+                    hearingOrdersBundleBefore.toBuilder().orders(caseManagementOrdersAfter).build())
+            ))
+            .build();
+
+        underTest.sendNotification(
+            new DraftOrdersRemovedEvent(
+                caseDataAfter,
+                caseDataBefore,
+                ORDER_TO_BE_REMOVED,
+                REMOVAL_REASON));
+
+        verify(notificationService).sendEmail(
```

Comment:
is this verifying that sendEmail isn't called?

Reply:
It is verifying that only the local authority email address is present and that the null returned by the courtService.getCourtEmailNotCtsc is not in the set of email addresses